src/gui/log/logview.h

```diff
@@ -26,35 +26,34 @@
  * exception statement from your version.
  */
 
-#ifndef LOGLISTWIDGET_H
-#define LOGLISTWIDGET_H
+#pragma once
+
+#include <QTreeView>
 
-#include <QListWidget>
 #include "base/logger.h"
 
-class QKeyEvent;
+class LogModel;
+class LogFilterModel;
 
-class LogListWidget : public QListWidget
+class LogView : public QTreeView
```

Comment:
To be consistent with Qt naming scheme it should be **LogWidget**. **View** is a widget that display content of model in a specific way (e.g. QTreeView, QListView etc.). When some view is bound with model then it has Widget suffix (e.g. QListWidget, QTreeWidget).

Reply:
OK. I understand. Will be renamed.